markdown/blog/helsinki-and-community.md

```diff
@@ -0,0 +1,87 @@
+
+## AsyncAPI Conf on Tour at APIDays Helsinki 2024
+
+Our next stop was [APIDays Helsinki 2024](https://www.apidays.global/helsinki_and_north/).
+
+Hugo kicked things off with a great introduction to the AsyncAPI spec and later wrapped up the track with an excellent session on testing event-driven architectures.
+
+<Figure
+  src="/img/posts/helsinki-and-community/hugo.webp"
+  caption="Hugo presenting AsyncAPI basics."
+  className="text-center"
+/>
+
+Jonas followed with a detailed overview of the AsyncAPI tooling ecosystem, showcasing the various tools built around the specification to support developers working with AsyncAPI documents.
+
+<Figure
+  src="/img/posts/helsinki-and-community/jonas.webp"
+  caption="Jonas presenting the AsyncAPI tooling ecosystem."
+  className="text-center"
+/>
+
+I then presented an overview of AsyncAPI use cases, demonstrating how different companies are using AsyncAPI in production, all based on real data we’ve gathered over the years. You can explore all of this in our [AsyncAPI case studies](/casestudies).
+
+<Figure
+  src="/img/posts/helsinki-and-community/lukasz.webp"
+  caption="Lukasz presenting how AsyncAPI is used in production."
+  className="text-center"
+/>
+
+The talks were engaging, and the questions that followed were great. But the best conversations, the most technical and insightful ones, happened at the `API Specs and Standards` booth. We teamed up with friends from the OpenAPI Initiative, and together we had deep, meaningful discussions.
+
+<Figure
+  src="/img/posts/helsinki-and-community/booth-front.webp"
+  caption="OpenAPI and AsyncAPI experts side by side. From left: Lukasz, Erik, Frank, Hugo, and Jonas."
+  className="text-center"
+/>
+
+## September 18-19 in London
+
+I won’t be attending the London event this year, but plenty of other folks will, especially at the `API Specs and Standards` booth. I hope you all have as much fun there as we did in Helsinki!
+
+<Figure
+  src="/img/posts/helsinki-and-community/run.webp"
+  caption="OpenAPI and AsyncAPI experts getting to know each other during a Helsinki city run."
+  className="text-center"
+/>
+
+If you haven’t grabbed your ticket yet, AsyncAPI community members can get free passes. Just [check out the agenda and reserve your spot](https://conference.asyncapi.com/venue/London).
+
+## December in Paris and the future of the Booth
+
+Looking ahead to December, we’ll have some free passes for the Paris event too. But honestly, the best experience is joining as a speaker. Not only do you get a free pass, but you also get to attend the speaker’s party the night before the event. Most importantly, it’s a unique opportunity to share your story with the wider [AsyncAPI](https://conference.asyncapi.com/venue/Paris) community. CfP is still open.
+
+Of course, we’ll have a booth again. This time, it won’t just be AsyncAPI and OpenAPI folks—our friends from JSON Schema will join too.
```

Comment:
```suggestion
Of course, we’ll have a booth again. This time, it won’t just be AsyncAPI and OpenAPI folks—our friends from JSON Schema will join, too.
```

Reply:
why comma before `too`?

Reply:
It's how I read the sentence and how it sounded, so I put a comma as an emphasis. It is still correct to have it there or not; it is the personal preference of the reviewer (me😉)